src/lightning_app/utilities/packaging/app_config.py

```diff
@@ -15,10 +15,12 @@ class AppConfig:
 
     Args:
         name: Optional name of the application. If not provided, auto-generates a new name.
+        checkpoint: Optional checkpoint to load the application from.
     """
 
     name: str = field(default_factory=get_unique_name)
     cluster_id: Optional[str] = field(default=None)
+    checkpoint: Optional[str] = None  # TODO: might be moved to appspec.
```

Comment:
What are the considerations here? Is it really needed to mention this in the source code?

Reply:
let's remove it